samples/model-builder/conftest.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google LLC
+# Copyright 2022 Google LLC
```

Comment:
The year change is only needed when adding a new file.

Reply:
Fixed